warehouse/templates/index.html

```diff
@@ -33,7 +33,7 @@ <h3 class="title"><a href="{{ request.route_path('packaging.project', name=relea
 {% block content %}
 <section class="banner-block">
   <div class="homepage-hero">
-    <h1 class="title" {{ l20n("homepageIntroduction") }}>Find, install and publish Python packages with the Python Package Index</h1>
+    <h1 class="title" {{ l20n("homepageIntroduction") }}>Find, install and publish Python projects with the Python Package Index</h1>
```

Comment:
Same here, I think the tag line works better as "packages", for the reasons Nick expounded in the original issue.